CMP1903_A1_2324/Testing.cs

```diff
@@ -16,5 +17,31 @@ internal class Testing
          */
 
         //Method
+        public void Test()
+        {
+            // Runs a test versio of the game, and checks the outputs of the rolls
+            Game testGame = new Game();
+            testGame.ExecuteGame();
+
+            if (testGame.TotalOfRolls() < 18)
+            {
+                if (testGame.TotalOfRolls() > 3)
+                {
+                    Console.WriteLine("Total Is As Expected: " + testGame.TotalOfRolls());
+                }
+                else
+                {
+                    Console.WriteLine("Total Was Less Than 3, Which Is Impossible: " + testGame.TotalOfRolls());
+                }
+            }
+            else
+            {
+                Console.WriteLine("Total Was Greater Than 18, Which Is Impossible: " + testGame.TotalOfRolls());
+            }
+            
```

Comment:
Use of error checking statements although, haven't used Debug.Assert() as in assessment brief

Reply:
I have now changed the code to include Debug.Assert() to check the game runs before running the main game, thank you for spotting this!